### ISSUE TITLE
tensor筛选

### ISSUE THREAD

**Issue by wxl1351641822** [PaddlePaddle/Paddle#29759]
想要问一下，paddle是否有类似torch或者numpy那种列表筛选的功能？
`active_loss=attention_mask_label.reshape(-1)==1
            active_logits=fluid.layers.softmax(fluid.layers.reshape(pred, shape=[-1, self.num_classes]))
            active_logits=active_logits.numpy()[active_loss]
            labels=labels.reshape(-1)[active_loss].reshape(-1,1)
            loss = fluid.layers.cross_entropy(to_variable(active_logits), to_variable(labels),ignore_index=0)`
activate_loss是个true,false的向量
然后active_logits=active_logits.numpy()[active_loss]，就过滤掉为False的行
--这样写的话，loss回传是不是会有问题，想问下paddle本身是否有类似的处理？

**Comment by sandyhouse** [comment]
paddle暂不支持这种处理。

**Comment by wxl1351641822** [comment]
那我转到numpy再转回来，这种写法是否回传会有问题，以及有列向量乘矩阵对应行相乘吗

**Comment by sandyhouse** [comment]
转换会导致backward出问题。没太理解『有列向量乘矩阵对应行相乘吗』什么意思？

**Comment by wxl1351641822** [comment]
>>> a = torch.ones(3,4)
>>> a
tensor([[1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.]])
>>> b = torch.Tensor([1,2,3]).reshape((3,1))
>>> b
tensor([[1.],
        [2.],
        [3.]])
>>> a * b
tensor([[1., 1., 1., 1.],
        [2., 2., 2., 2.],
        [3., 3., 3., 3.]])

这个样子

**Comment by wxl1351641822** [comment]
`>>> a = torch.ones(3,4)
>>> a
tensor([[1., 1., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.]])
>>> b = torch.Tensor([1,2,3]).reshape((3,1))
>>> b
tensor([[1.],
        [2.],
        [3.]])
>>> a * b
tensor([[1., 1., 1., 1.],
        [2., 2., 2., 2.],
        [3., 3., 3., 3.]])
'

**Comment by sandyhouse** [comment]
paddle.multiply

**Comment by wxl1351641822** [comment]
额，paddle 1.8.5有multiply吗

**Comment by wxl1351641822** [comment]
![image](https://user-images.githubusercontent.com/42665925/102607654-973f2e00-4163-11eb-8a33-418b881a7d0d.png)
你看他的标识……我这边找不到这个orz

**Comment by sandyhouse** [comment]
1.8.2版本可以使用paddle.fluid.layers.elementwise_mul
